lvsfunc/deblock.py

```diff
@@ -2,21 +2,22 @@
     Deblocking functions.
 """
 from functools import partial
-from typing import List, Optional, Sequence, Tuple, cast
+from typing import Optional, Sequence, Tuple
 
 import vapoursynth as vs
-from vsutil import scale_value
 
 from .types import Matrix
+from .util import get_prop
 
 core = vs.core
 
 
 def autodb_dpir(clip: vs.VideoNode, edgevalue: int = 24,
-                strs: Sequence[float] = (30, 50, 75),
+                strs: Sequence[float] = [30, 50, 75],
                 thrs: Sequence[Tuple[float, float, float]] = [(1.5, 2.0, 2.0), (3.0, 4.5, 4.5), (5.5, 7.0, 7.0)],
                 matrix: Optional[Matrix] = None,
                 cuda: bool = True, device_index: int = 0,
+                write_props: bool = False
```

Comment:
Is there a reason to make this a param rather than always add it to the clip?

Reply:
There are already a lot of props per frame. Adding more of them would seem unreadable.

Reply:
I think it'll be fine. Not a whole lot of filters/functions actually seem to add props at all, so it will probably still be readable to most users.

Reply:
Also, I didn't benchmark it but it's still more logic into a FrameEval function. My way of doing things is always avoid the more possible logic in FrameEval function since they can relatively slow down the processing.